lib/foods.scss

```diff
@@ -1,5 +1,58 @@
-.foods-table {
-  font-size: 14px;
+%format-table {
+  width: 300px;
+  display: flex;
+  justify-content: space-between;
+}
+
+%format-row {
+  height: 25px;
+  margin: 0px;
+  display: flex;
+  align-items: center;
+}
+
+.food-headers {
+  @extend %format-table;
+  @extend %format-row;
+
+  .name-header {
+    width: 200px;
+  }
+
+  .calories-header {
+    width: 60px;
+  }
+
+  .hidden {
+    width: 50px;
+  }
+}
+
+#food-item-row {
+  @extend %format-table;
+  @extend %format-row;
+
+  .food-item-name {
+    width: 200px;
+  }
+
+  .food-item-calories {
+    width: 50px;
+  }
+
+  .button-container {
+    width: 25px;
+
+    .food-item-delete-btn {
```

Comment:
When I saw the initial screenshot, I thought we had included an image as the button. 

Really cool to see how to make the button with styling only.